dom.bs

```diff
@@ -832,34 +823,27 @@ using <var>eventInterface</var>, which must be either {{Event}} or an interface
 it, and optionally given a <a>Realm</a> <var>realm</var>, run these steps:</p>
 
 <ol>
- <li>
-  <p>Create a new object <var>event</var> using <var>eventInterface</var>. If <var>realm</var>
-  is given, use that Realm; otherwise, use the default behavior defined in Web IDL.
-
-  <p class="XXX">As of the time of this writing Web IDL does not yet define any default behavior;
-  see <a href="https://github.com/heycam/webidl/issues/135">heycam/webidl#135</a>.
-
- <li><p>Set <var>event</var>'s <a>initialized flag</a>.
+ <li><p>If <var>realm</var> is not given, then set it to null.
 
- <li><p>Let <var>defaultEventInitDict</var> be the result of
- <a lt="converted to an IDL value">converting</a> the JavaScript value undefined to the dictionary
- type accepted by <var>eventInterface</var>'s constructor. (This dictionary type will either be
- {{EventInit}} or a dictionary that inherits from it.)
+ <li>
+  <p>Let <var>dictionary</var> be the result of <a lt="converted to an IDL value">converting</a>
+  the JavaScript value undefined to the dictionary type accepted by <var>eventInterface</var>'s
+  constructor. (This dictionary type will either be {{EventInit}} or a dictionary that inherits from
+  it.)
 
- <li>For each <a>dictionary member</a> present in <var>defaultEventInitDict</var>, find the
- attribute on <var>event</var> whose <a spec=webidl>identifier</a> matches the key of the
- <a>dictionary member</a> and then set the attribute to the default value of that
- <a>dictionary member</a>.
+  <p class="XXX">This does not work if members are required; see
+  <a href="https://github.com/whatwg/dom/issues/600">whatwg/dom#600</a>.
 
- <li><p>Set <var>event</var>'s {{Event/timeStamp}} attribute to a {{DOMHighResTimeStamp}}
- representing the high resolution time from the <a>time origin</a> to the occurrence that the
- event is signaling.
+ <li>
+  <p>Let <var>event</var> be the result of running the <a>inner event creation steps</a> with
+  <var>eventInterface</var>, <var>realm</var>, the occurrence that the event is signaling, and
+  <var>dictionary</var>.
 
- <p class=note>For example, in macOS the occurrence time for input actions are available via the
- timestamp property of corresponding <code>NSEvent</code> objects. So in this case, the
- {{Event/timeStamp}} value will be equivalent to the <code>NSEvent</code>'s timestamp offset by
- <a>time origin</a>, translated in milliseconds, and with its precision adjusted to meet the minimum
- <a>clock resolution</a>.
+  <p class=note>For example, in macOS the occurrence time for input actions are available via the
```

Comment:
...occurrence time for input actions **is** available via the timestamp property of **the** corresponding NSEvent **object**.

(or  ...occurrence **times** for input actions are available via the timestamp property of **the** corresponding NSEvent objects.)